doc/authors.md

```diff
@@ -1,23 +1,23 @@
 Zcash Contributors
```

Comment:
Shal we leave 'zcash' ?

Reply:
Let's leave it for now, for the release after this we can change the script that auto-generates authors.md.